Missing math.h include

The application fails to compile because `rb_sensor_monitor.c` needs to include `math.h`.

<details>

```
gcc -MD -MP -DNDEBUG -std=gnu99 -D_GNU_SOURCE -Wunused -Wcast-align -Wdisabled-optimization -Winit-self -Wlogical-op -Wmissing-include-dirs -Wstrict-overflow=5 -Wundef -I. -Isrc/ -g -O2 -Wall -Wsign-compare -Wfloat-equal -Wpointer-arith -DNDEBUG  -c src/rb_array.c -o src/rb_array.o
gcc -MD -MP -DNDEBUG -std=gnu99 -D_GNU_SOURCE -Wunused -Wcast-align -Wdisabled-optimization -Winit-self -Wlogical-op -Wmissing-include-dirs -Wstrict-overflow=5 -Wundef -I. -Isrc/ -g -O2 -Wall -Wsign-compare -Wfloat-equal -Wpointer-arith -DNDEBUG  -c src/rb_sensor_monitor.c -o src/rb_sensor_monitor.o
src/rb_sensor_monitor.c: In function ‘rb_monitor_op_value0’:
src/rb_sensor_monitor.c:497:8: warning: implicit declaration of function ‘isnormal’ [-Wimplicit-function-declaration]
   if (!isnormal(number)) {
        ^
src/rb_sensor_monitor.c: In function ‘process_vector_monitor’:
src/rb_sensor_monitor.c:854:9: warning: implicit declaration of function ‘isfinite’ [-Wimplicit-function-declaration]
     if (isfinite(result)) {
         ^
In file included from src/rb_sensor_monitor.c:25:0:
```

</details>

<details>

```
gcc -DNDEBUG -std=gnu99 -D_GNU_SOURCE -Wunused -Wcast-align -Wdisabled-optimization -Winit-self -Wlogical-op -Wmissing-include-dirs -Wstrict-overflow=5 -Wundef -I. -Isrc/ -g -O2 -Wall -Wsign-compare -Wfloat-equal -Wpointer-arith -DNDEBUG  src/main.o src/rb_snmp.o src/rb_value.o src/rb_zk.o src/rb_monitor_zk.o src/rb_sensor.o src/rb_sensor_queue.o src/rb_array.o src/rb_sensor_monitor.o src/rb_sensor_monitor_array.o src/rb_message_list.o src/rb_libmatheval.o -o rb_monitor -lrd -lpthread -lz -lrt -lpcap -lpthread -ljson-c -lrdkafka -lpthread -lz -lrd -lrt -lz -lm -lmatheval -lsnmp
src/rb_sensor_monitor.o: En la función `rb_monitor_op_value0':
/home/diego/Documentos/src/github.com/redBorder/rb_monitor/src/rb_sensor_monitor.c:497: referencia a `isnormal' sin definir
src/rb_sensor_monitor.o: En la función `process_vector_monitor':
/home/diego/Documentos/src/github.com/redBorder/rb_monitor/src/rb_sensor_monitor.c:854: referencia a `isfinite' sin definir
collect2: error: ld returned 1 exit status
```

</details>

--- a/src/rb_sensor_monitor.c
+++ b/src/rb_sensor_monitor.c
@@ -27,6 +27,7 @@
 #include "rb_json.h"
 
 #include <matheval.h>
+#include <math.h>
 #include <librd/rdfloat.h>
 
 static const char DEFAULT_TIMESTAMP_SEP[] = ":";

